claraabk/projetoIP
# (feature):release_develop_branch

## What I did

- Add project structure design
- Add requirements.txt
- Implement Game Loop class
  - run method
  - quit method
  - draw method
  - settings file

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   a. `git pull`
   b. `git checkout develop`
   c. `git pull origin develop`
   
2. In VScode open `terminal`
   a.  open your python venv
   b. `python -m Game` or `python run.py`

3. Things to do during the code:

   - Assess whether the new structure is suitable for the project.
   - Make sure the code is understandable and has low complexity.

4. Read the changed files section on GitHub

5. Test the things that you think are worth testing, even the ones that are not in this description

&nbsp;

---

**Time spent:** 2 hrs 20 mins
